build/testBuildImages.sh

```diff
@@ -12,6 +12,13 @@ declare -r testProjectName="Oryx.BuildImage.Tests"
 
 # Load all variables
 source $REPO_DIR/build/__variables.sh
+source $REPO_DIR/build/__sdkStorageConstants.sh
+
+if [ -z "$ORYX_TEST_SDK_STORAGE_URL" ]; then
+    echo
+    echo "Setting environment variable 'ORYX_TEST_SDK_STORAGE_URL' to default value '$DEV_SDK_STORAGE_BASE_URL'."
```

Comment:
nit: since this message is repeated in a few spots, it would be nice for clarity to add in _which_ test suite this environment variable is being set for.

"Setting environment variable 'ORYX_TEST_SDK_STORAGE_URL' to default value '$DEV_SDK_STORAGE_BASE_URL' _for build image tests_."